scripts/root_dir.mjs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import path from "path";
+import url from "url";
+
+// WARNING: if you move this file, you MUST update this file path
+const RELATIVE_PATH_FROM_THIS_FILE_TO_PROJECT_ROOT = "../..";
+
+export function rootDir(fromLocation = process.argv[1]) {
```

Comment:
process.argv[1] should always be the correct absolute system path to the script being run, which starts with the drive letter on windows and the leading slash in linux/unix. it's not great, but in cases where that doesn't work, we can override it.

Reply:
The problem with `URL` is that it adds in the leading slash on windows, which is incorrect.

Reply:
Ah, we're not using URL.pathname anymore

Reply:
it actually isn't doing much now:

```
export function getRootDir(fromLocation = process.argv[1], pathToRoot = RELATIVE_PATH_FROM_THIS_FILE_TO_PROJECT_ROOT) {
  return path.resolve(fromLocation, pathToRoot);
}
```

maybe we just `path.resolve()` instead